examples/flat-serialisation/anonymous-owner.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
I don't think this aligns with Option 1 from #57 so we should remove these from the file. 

Ideally the examples directory should include a full valid statement package.